pkg/operator/operatorclient/operatorclient.go

```diff
@@ -34,12 +36,17 @@ func (c OperatorClient) Informer() cache.SharedIndexInformer {
 	return c.Informers.Operator().V1alpha1().CertManagers().Informer()
 }
 
+// GetOperatorState returns the Operator state. If it's missing (e.g. the certmanager CR hasn't been creates),
+// it will create a default one.
 func (c OperatorClient) GetOperatorState() (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, string, error) {
 	instance, err := c.Informers.Operator().V1alpha1().CertManagers().Lister().Get("cluster")
+	if apierrors.IsNotFound(err) {
+		//TODO: Consider using propagated context here but this requires changes in library-go. Maybe it's not worth it?
+		instance, err = c.createDefaultCertManager(context.TODO())
```

Comment:
As discussed I believe we have the open question how this can be removed at uninstall time.

Reply:
If you're uninstalling the Operator? I think it will be removed by OLM, but let me double check this scenario.